nautobot/users/tests/test_views.py

```diff
@@ -91,3 +91,29 @@ def test_change_password_disabled(self):
                     "Remotely authenticated user credentials cannot be changed within Nautobot.",
                     str(response.content),
                 )
+
+
+class AdancedProfileSettingsViewTest(TestCase):
+    """
+    Tests for the user's advanced settings profile edit view
+    """
+
```

Comment:
Would be good to also test authn and authz of this view.

Reply:
Do we have an example of what you are thinking here? This was mostly copy-pasta from the change password test cases in the same file.

Reply:
Or are you talking about the superuser access of the silk UI?

Reply:
`AdvancedProfileSettingsEditView` already has the `LoginRequiredMixin` on it, but I will add a test for the silk UI.